src/pages/Main/Slider.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import ImgComp from './ImgComp';
+import i1 from './SlideData/1.png';
+import i2 from './SlideData/2.png';
+import i3 from './SlideData/3.png';
+import i4 from './SlideData/4.png';
+import i5 from './SlideData/5.png';
+
+function Slider() {
+  let sliderArr = [
+    <ImgComp src={i1} key />,
+    <ImgComp src={i2} key />,
+    <ImgComp src={i3} key />,
+    <ImgComp src={i4} key />,
+    <ImgComp src={i5} key />,
+  ];
+
+  const [x, setX] = useState(0);
+
+  const goLeft = () => {
+    x === 0 ? setX(-100 * (sliderArr.length - 1)) : setX(x + 100);
+  };
+
+  const goRight = () => {
+    x === -100 * (sliderArr.length - 1) ? setX(0) : setX(x - 100);
+  };
+
+  return (
+    <Container>
+      {sliderArr.map((item, index) => {
+        return (
+          <Slide key={index} style={{ transform: `translateX(${x}%)` }}>
+            {item}
+          </Slide>
+        );
+      })}
```

Comment:
컴포넌트 자체를 맵돌리는 것이 아니라, 데이터를 맵돌려 컴포넌트에 props로 데이터를 넘겨주세요.
```suggestion
      {sliderArr.map((item, index) => {
        return (
          <Slide key={index} style={{ transform: `translateX(${x}%)` }}>
            <ImgComp src={item} /> // item이 이미지 주소가 되게 sliderArr 작성
          </Slide>
        );
      })}
```

Reply:
완저히 맵돌맀씁니다!